docs/addons/kube-state-metrics.md

```diff
@@ -0,0 +1,48 @@
+# Kube State Metrics Add-on
+
+This add-on installs [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics).
+
+kube-state-metrics (KSM) is a simple service that listens to the Kubernetes API server and generates metrics about the state of the objects. (See examples in the Metrics section below.) It is not focused on the health of the individual Kubernetes components, but rather on the health of the various objects inside, such as deployments, nodes and pods.
+
+kube-state-metrics Add-on is about generating metrics from Kubernetes API objects without modification. This ensures that features provided by kube-state-metrics have the same grade of stability as the Kubernetes API objects themselves.
+
+## Usage
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.KubeStateMetricsAddOn()
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options

```

Comment:
If I am a customer and I am reading this section my impression will be that the addon supports just two attributes. I do think we need to call out standard configuration options (helm repo, chart, version, etc.) however it should not be under values:

- `createNamespace`: ...
- `values`: ...
- [standard helm configuration options](./index.md#standard-helm-add-on-configuration-options).

Applies elsewhere.

Reply:
Fixed

Reply:
Is it? Don't see the change.